vsintegration/tests/Salsa/VisualFSharp.Salsa.fsproj

```diff
@@ -8,6 +8,7 @@
     <NoWarn>$(NoWarn);45;47;52;58;75</NoWarn>
     <DisableImplicitFSharpCoreReference>true</DisableImplicitFSharpCoreReference>
     <UsePackageTargetFallbackHack>true</UsePackageTargetFallbackHack>
+    <EnableDefaultEmbeddedResourceItems>false</EnableDefaultEmbeddedResourceItems>
```

Comment:
What's the benefit, particularly with this project and `VisualFSharp.UnitTests` where there are no `*.resx` files?  If the `**/*.resx` glob is breaking incremental build then that's a larger issue we should address at the SDK level.

Reply:
@brettfo VisualFSharp.UnitTests has projects in sub-directories which put resources in `obj/Debug` directories when built, and the implicit glob was picking those up

Reply:
I didn't know about the `obj\Debug` issue.  I see that you rearranged the directories which I like.  Only one final nit-pick; instead of decorating each project with `<EnableDefault...>false</>` can you add it to an appropriate upstream `Directory.Build.props` or `FSharp.Directory.Build.props`?  That way we won't forget to add it to any new projects in those subdirectories.